### PR TITLE
Adding JarScanner to context.xml

### PR DESCRIPTION
**Pull request by mkelderm**

### CODE REVIEW

File: manifests/context.pp [hunk 1]
```diff
@@ -150,6 +151,16 @@
     }
   }
 
+  # Template uses:
+  # - $jarscanner
+  if $jarscanner and $jarscanner != [] {
+    concat::fragment { "${name} tomcat context jarscanner":
+      order   => 030,
```
Comment (antoineco) [inline manifests/context.pp:158]: The order should be `080` if you want it to be added after resourcelinks.

File: manifests/context.pp [hunk 2]
```diff
@@ -150,6 +151,16 @@
...
+      content => template("${module_name}/common/context.xml/015_jarscanner.erb"),
```
Comment (antoineco) [inline manifests/context.pp:159]: The file prefix should match the value of the `order` parameter.

File: manifests/config.pp [hunk 3]
```diff
@@ -327,6 +328,7 @@
     realm            => $context_realm,
     resources        => $context_resources,
     watchedresources => $context_watchedresources,
+    jarscanner       => $context_jarscanner,
```
Comment (antoineco) [inline manifests/config.pp:331]: Should be moved after `resourcelinks` in order to keep a consistent order.

**Review by antoineco — CHANGES_REQUESTED** [review]
A few minor details to fix, looks good overall.

**Comment by mkelderm** [comment]
I am fine with it.

**Pull request closed without merging**